tests/spec/core/caniuse-spec.js

```diff
@@ -0,0 +1,133 @@
+"use strict";
+describe("Core — Can I Use", function () {
+  afterAll(flushIframes);
+
+  it("adds proper placeholders", async () => {
+    const ops = makeStandardOps({ caniuse: "FEATURE" });
+    const doc = await makeRSDoc(ops);
+
+    const title = doc.getElementById("caniuse-FEATURE");
+    const placeholder = title.nextElementSibling;
+    expect(title.textContent).toBe("Can I Use this API?");
+    expect(placeholder.textContent).toBe("Fetching data from caniuse.com...");
+    expect(placeholder.localName).toBe("dd");
+  });
+
+  it("uses meaningful defaults", async () => {
+    const ops = makeStandardOps({ caniuse: "FEATURE" });
```

Comment:
For most of these, we probably want to override with `apiURL`, otherwise, we just end up with a bunch of 404s. However, if we want to do a "real" run, we can grab a well-known feature here.